lib/stat_tracker.rb

```diff
@@ -31,54 +31,23 @@ def self.from_csv(info)
     def self.game_factory(info)
         games = []
         CSV.foreach(info[:games], headers: true, header_converters: :symbol) do |row|
-            class_info = {:game_id => row[:game_id],
-            :season => row[:season],
-            :type => row[:type],
-            :date_time => row[:date_time],
-            :away_team_id => row[:away_team_id],
-            :home_team_id => row[:home_team_id],
-            :away_goals => row[:away_goals],
-            :home_goals => row[:home_goals],
-            :venue => row[:venue],
-            :venue_link => row[:venue_link]}
-            games << Game.new(class_info)
+            games << Game.new(row)
```

Comment:
This is looking so clean and fantastic. Refactoring like this is clean and more dynamic and very DRY